charts/incubator/wordpress/questions.yaml

```diff
@@ -59,6 +59,20 @@ questions:
             type: string
             required: true
             default: Truecharts Blog
+        - variable: WORDPRESS_ENABLE_HTTPS
+          label: Wordpress Enable HTTPS support
+          description: Whether to use HTTPS by default
+          schema:
+            type: string
+            required: true
+            default: no
```

Comment:
Default should be false

Reply:
https://github.com/bitnami/containers/tree/main/bitnami/wordpress#user-and-site-configuration

They say "no", but I mean is that mean "false" or "no" lol

Reply:
Oof my mistake, that's a string.

Reply:
Try using an enum instead for yes and no values

Reply:
i'm very very basic so I still question things myself